src/persist/benches/end_to_end.rs

```diff
@@ -11,82 +11,51 @@
 //! we can ingest it.
 
 use std::path::{Path, PathBuf};
-use std::time::Duration;
 
 use criterion::measurement::Measurement;
+use criterion::{criterion_group, criterion_main, Bencher, BenchmarkId, Criterion, Throughput};
 use differential_dataflow::operators::Count;
 use differential_dataflow::AsCollection;
-use persist::error::Error;
+use ore::cast::CastFrom;
+use ore::metrics::MetricsRegistry;
 use timely::dataflow::operators::Map;
 use timely::dataflow::ProbeHandle;
 use timely::Config;
 
-use ore::metrics::MetricsRegistry;
-
-use persist::file::{FileBlob, FileLog};
-use persist::indexed::runtime::{self, RuntimeClient, RuntimeConfig, StreamWriteHandle};
+use persist::error::{Error, ErrorLog};
+use persist::file::FileBlob;
+use persist::indexed::runtime::{self, RuntimeClient, RuntimeConfig};
 use persist::operators::source::PersistedSource;
 use persist::storage::LockInfo;
-
-use criterion::{criterion_group, criterion_main, Bencher, BenchmarkGroup, Criterion, Throughput};
-use rand::distributions::Alphanumeric;
-use rand::Rng;
+use persist::workload::{self, DataGenerator};
 
 pub fn bench_end_to_end(c: &mut Criterion) {
-    let data_size_mb = 100;
-    let chunk_size_mb = 5;
-
     let mut group = c.benchmark_group("end_to_end");
-    group.throughput(Throughput::Bytes((data_size_mb as u64) * 1024 * 1024));
-
-    write_and_bench_read(&mut group, "4b_keys", data_size_mb, chunk_size_mb, 4, 0);
```

Comment:
the various flavors of this are now covered by setting the env vars, if we decide we need them